sql-plugin/src/main/java/com/nvidia/spark/rapids/GpuColumnVector.java

```diff
@@ -472,7 +472,7 @@ private static DType toRapidsOrNull(DataType type) {
       return DType.TIMESTAMP_DAYS;
     } else if (type instanceof TimestampType) {
       return DType.TIMESTAMP_MICROSECONDS;
-    } else if (type instanceof StringType) {
+    } else if (type instanceof StringType || type instanceof BinaryType) {
```

Comment:
We have a decision to make. Is BinaryType the same as a DType.STRING, or is it the same as a DType.LIST of DType.INT8 where the bytes are not nullable?  Internally the data layout will be identical.  But to me Binary can store things that are technically illegal to be stored in a String.  String type data is encoded as UTF8.  Some characters require a multi-byte encoding and it is technically an error to only store the first bytes of a multi-byte encoded string.  I personally wold rather store it as a LIST of INT8.  But I am open to arguments as to why it is better to store it as a STRING instead.

Reply:
`DType.LIST` of `DType.INT8` (CHAR if available?) makes more sense to me, as in C++ people frequently use `vector<char>` to store binary.

Reply:
I agree with @ttnghia. And I changed the underlying DType of BinaryType to LIST.